itests/docker/docker.go

```diff
@@ -175,14 +179,27 @@ func (d *Docker) Finish(cancel context.CancelFunc) {
 	cancel()
 }
 
+func (d *Docker) stopContainer(containerID string) error {
+	const shutdownTimeout = 5 // In seconds.
+	if stErr := d.pool.Client.StopContainer(containerID, shutdownTimeout); stErr != nil {
```

Comment:
I think `StopContainer` internally sends to the running application `SIGINT` of `SIGTERM`. Or not? Because I think if case of stop error container should be killed with `SIGKILL`.

Reply:
Yes, StopContainer sends SIGTERM in no option given. 

> If no signal is configured for the container, SIGTERM is used as default.

I agree, that it will be better to send SIGKILL to kill containers in case of stop failure.